### ISSUE TITLE
erroneously cuts at newline??

### ISSUE THREAD

**Issue by yannickwurm** [wurmlab/genevalidatorapp#10]
The following two sequences are not treated identically:

```
>test
MLPQTPDIIPTTLNQQKCVKARALYDNIAEAPDELAFRKGDVLTVLEQNTAGLEGWWLCALRGRQGICPGNRLRLLVGQYDTGGCLVGSRPDLTISEDGIQRHGKRRSWHVQPNRVVTPQKCGDVYLYDLPASRGSPAPPSRHDSPLNSNNEHLHNSGRYTTSSRNSVDNGGDVSDCYDVPPRAIPVIPSPASSPSPAPSCYDIPRPPTSCTPISNCSGGSGVTPLDCYDVPRPLQPLTPSSSASSLTNDGSLSGSNRSSLAAPDYDVPRSRLPASSLPSRHNTPVPKTPTPPPPPQTQQIYDVPVSKELPLELDSALEGLQRLQSEASAAIARLLGFVSPVWRTPQRLDATLMDLRLAALRLRTSLHDLAEFAEGTLGNAGKAPDKGLATKLRPLVKALRDSDKLVQEAATELDAMEWDAGKLCRGGGDTPTPTNGPPSTILPPAQPDPLDQLIACARALTEDVRQVASFIQGNSTLLFKRSSIISTGSSNNSGAGEDYDYVNLDSREVVAKQREEVRASLPQELRSNYDLLVSESDNATIQMPPTTPTPMDPNDKQLLAFYAAQVITHGNHLTHAIDAFMQTVEHNQPPKVFLAHGKFVVLSAHRLVHIGDTVHRNVIRNDVKTRVLECANALNEALKQTVSKTKQAAQFFPSVSAVQEMVDSVVDVSHLAKDLKVAIINGAQQPMEVTSSNFQEVLVELDEILKNATFLCIDGEFTGLNSGPDGGVFDTPAQYYAKLRTGSMDFLLIQFGLSVFTFNKEMQKYNQRSYNFYVFPRPLNRMAPDCRFMCQTSSISFLASQGFDFNKLFKLGIPYLTTNEEEKLMKRLEEKQRIRDEGTEILPISDVERPQIEEICSRIDEFVTSETEELLIEKCNAFIRRLVYQEVKLRWPNKLKVESKMNNFGCILVVQRLGTKEEEEQREIEKREREKTEIQQAVGLSILMRKIADSGKLIVGHNMLLDLCHIVHQFFGQLPESYFEFKSLVHSLFPRILDTKIICHSQQFKENIPSSNLGILLETVSKSPFKITEVEPIDGRSYSTLSEKCHEAGYDAYITGICFIALSNYLGSLQKPEVPIVLSDSPLLNPFLNKLLIARLKDVPYINLVGDDPNPSRDHVFHLTFPKEWKFNDISHLFSPFGSVHVSWLSDISAYIELHRRDQVNEVMKVLAKTSTYKLQRYADYQASLENFNTGERKRKLSSSEETTPEAEELCGCRECAKIETETLCRA
```

Length for the following one is only 70. 

```
>test2
MLPQTPDIIPTTLNQQKCVKARALYDNIAEAPDELAFRKGDVLTVLEQNTAGLEGWWLCALRGRQGICPG
NRLRLLVGQYDTGGCLVGSRPDLTISEDGIQRHGKRRSWHVQPNRVVTPQKCGDVYLYDLPASRGSPAPP
SRHDSPLNSNNEHLHNSGRYTTSSRNSVDNGGDVSDCYDVPPRAIPVIPSPASSPSPAPSCYDIPRPPTS
CTPISNCSGGSGVTPLDCYDVPRPLQPLTPSSSASSLTNDGSLSGSNRSSLAAPDYDVPRSRLPASSLPS
RHNTPVPKTPTPPPPPQTQQIYDVPVSKELPLELDSALEGLQRLQSEASAAIARLLGFVSPVWRTPQRLD
ATLMDLRLAALRLRTSLHDLAEFAEGTLGNAGKAPDKGLATKLRPLVKALRDSDKLVQEAATELDAMEWD
AGKLCRGGGDTPTPTNGPPSTILPPAQPDPLDQLIACARALTEDVRQVASFIQGNSTLLFKRSSIISTGS
SNNSGAGEDYDYVNLDSREVVAKQREEVRASLPQELRSNYDLLVSESDNATIQMPPTTPTPMDPNDKQLL
AFYAAQVITHGNHLTHAIDAFMQTVEHNQPPKVFLAHGKFVVLSAHRLVHIGDTVHRNVIRNDVKTRVLE
CANALNEALKQTVSKTKQAAQFFPSVSAVQEMVDSVVDVSHLAKDLKVAIINGAQQP
MEVTSSNFQEVLVELDEILKNATFLCIDGEFTGLNSGPDGGVFDTPAQYYAKLRTGSMDFLLIQFGLSVF
TFNKEMQKYNQRSYNFYVFPRPLNRMAPDCRFMCQTSSISFLASQGFDFNKLFKLGIPYLTTNEEEKLMK
RLEEKQRIRDEGTEILPISDVERPQIEEICSRIDEFVTSETEELLIEKCNAFIRRLVYQEVKLRWPNKLK
VESKMNNFGCILVVQRLGTKEEEEQREIEKREREKTEIQQAVGLSILMRKIADSGKLIVGHNMLLDLCHI
VHQFFGQLPESYFEFKSLVHSLFPRILDTKIICHSQQFKENIPSSNLGILLETVSKSPFKITEVEPIDGR
SYSTLSEKCHEAGYDAYITGICFIALSNYLGSLQKPEVPIVLSDSPLLNPFLNKLLIARLKDVPYINLVG
DDPNPSRDHVFHLTFPKEWKFNDISHLFSPFGSVHVSWLSDISAYIELHRRDQVNEVMKVLAKTSTYKLQ
RYADYQASLENFNTGERKRKLSSSEETTPEAEELCGCRECAKIETETLCRA

```

**Comment by yannickwurm** [comment]
(@ismailm this is a _major_ bug because most fastas are multiline)

**Comment by yannickwurm** [comment]
@yeban @IsmailM

**Comment by IsmailM** [comment]
Fixed - 5725c7483a9d8bc6b6c5487ff82e701ff25b1c4b

I'll release an updated gem shortly.
